### PR TITLE
Feature/lib upgrades

### PR DESCRIPTION
**Pull request by YaphetKG**
Adds more performant libraries for async io loops and http handling.

References:
https://github.com/MagicStack/uvloop#performance

https://github.com/python-hyper/h11/issues/9#issuecomment-242908968

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,13 +26,10 @@ RUN pip install -e .
 # gunicorn, hypercorn also options https://fastapi.tiangolo.com/deployment/manually/
```
Comment (PhillipsOwen) [inline Dockerfile:26]: there is a new base image that chris and i are trying to promote to get all the images at python 3.9.10. 

FROM renciorg/renci-python-image:v0.0.1

dockerfile at: https://github.com/TranslatorSRI/RENCI-Python-image

example usage at: https://github.com/TranslatorSRI/bl_lookup/blob/dockerfile_updates/Dockerfile

**Review by PhillipsOwen — CHANGES_REQUESTED** [review]
see note.

File: Dockerfile (hunk 1 above)
Comment (YaphetKG) [reply Dockerfile:21]: Updated the from tag of the image, and removed user creation from this docker file

**Review by PhillipsOwen — COMMENTED** [review]
looks good, thanks!

**Pull request merged**